src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -0,0 +1,32 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Delete this file. When someone creates a new servlet that they actually intend to use, thats when we'll add one